FlattenNextedListIterator.java

```diff
@@ -0,0 +1,77 @@
+//TC == ?? can you explain time and space complexity for this problem.
+
```

Comment:
Space will be N +D (Depth of the list) as stack will be having max size = deepest node/list from NestedList array

Reply:
Same for space, N+D in total, (N nodes/lists with Avg depth = D)